server/controllers/contact_list_controller.js

```diff
@@ -5,11 +5,52 @@ const path = require('path');
 const Settings = require('../../settings');
 const optIn = 'opt-in';
 
+function prepareNotificationEmail(reqBody) {
+	const subject = "New signup "
+	const url = formatUrl(Settings.url) + '/success';
+	const mailText = "Thanks for signing up. <a href='" + url + "'>Please click here to confirm your email address.</a>  This link will be active for 24 hours.";
+
+	var emailBody = {
+	  personalizations: [
+	    {
+	      to: [
+	        {
+	          email: reqBody.email,
+	        }
+	      ],
+	      subject: subject,
+	      custom_args: {
+	      	type: optIn,
+	      	time_sent: String(Date.now()),
+	      }
+	    },
+	  ],
+	  from: {
+	    email: Settings.senderEmail,
+	    name: Settings.senderEmail,
+	  },
+	  content: [
+	    {
+	      type: "text/html",
+	      value: mailText,
+	    }
+	  ],
+	}
+
+	const templateId = Settings.templateId;
+	if (templateId) emailBody.template_id = templateId;
+
+	for (key in reqBody) {
+		emailBody.personalizations[0].custom_args[key] = reqBody[key];
+	}
+
+	return emailBody;
+}
+
 function prepareEmail(reqBody) {
-	const subject = "Please Confirm Your Email Address";
+	const subject = reqBody.first_name;
```

Comment:
I think this was a mistake.  reqBody.first_name looks to only be relevant in the notification email.  Going to revert back.